island-is/island.is
# feat(samradsgatt): Kam 750 - pagination a malefnaspjoldum

# Add pagination to main page

https://veflausnir.atlassian.net/browse/KAM-750

## What

Added pagination to the main page
Added extra cases for testing

## Why

...

## Screenshots / Gifs

![image](https://user-images.githubusercontent.com/106164521/221815000-6e2a5e2d-605c-48ab-97bf-dabff13852ad.png)

## How to test
Checkout the branch feature/KAM-750-feature-pagination-a-malefnaspjoldum
Run the project locally
Scroll down to the bottom of the main page.
Observe and test the pagination.

## Checklist:
